app/scripts/controllers/reviewRegistration.js

```diff
@@ -68,6 +68,182 @@ angular.module('confRegistrationWebApp')
+          }
+          else {
+            $scope.spouseRegistration = {};
+            $scope.spouseExists = false;
+            $rootScope.includeSpouseButtonVisible = false;
+            console.log('No current and/or spouse registration found for user with ID.');
+          }
+        }).catch(function(response) {
+          $scope.spouseRegistration = {};
+          $rootScope.includeSpouseButtonVisible = false;
+          console.log('No current and/or spouse registration found for user with ID. Status = ' + response.status);
+      });
+    }
+
+    //Parameters needed to retrieve all current registrants for conference.id
+    var queryParameters = {
+      block: [],
+      //page: 1,
+      //limit: 20,
+      orderBy: 'last_name',
+      order: 'ASC',
+      filter: '',
+      filterPayment: '',
+      filterRegType: '',
+      includeCheckedin: 'yes',
+      includeWithdrawn: 'yes',
+      includeIncomplete: 'yes'
+    };
+
+    // Spouse Registration section visibility
+    $scope.spouseSectionVisible = {
+      spouse: false,
+      current: false,
+      both: false,
+      neither: false
+    };
+
+    //Get all current registrants based on conference.id and queryParameters
+    $http.get('conferences/' + conference.id + '/registrations', {params: queryParameters})
+      .then(function (response) {
+        // (spouseEmailExists = false) then show confirmation page
+        // (spouseEmailExists = true) then do not show confirmation page
+        // var spouseEmailExists;
+        // spouseEmailExists = _.intersection(_.map($scope.spouseRegistration.registrants, 'email'),
+        //     _.map($scope.currentRegistration.registrants, 'email')).length > 0;
+
+        var spouseRegistrantRegistered = false;
+        var currentRegistrantRegistered = false;
+        var spouseRegistrantCompleted = false;
+        var currentRegistrantCompleted = false;
+
+        //Using traditional for loop over angular.forEach because angular.forEach cannot break out of loops
+        //Also didn't think Lodash or underscore would be appropriate here either
+        //Check if spouse already registered
+        for (var i = 0, len = response.data.registrations.length; i < len; i++) {
+          spouseRegistrantCompleted = response.data.registrations[i].completed;
+          for (var i2 = 0, len2 = response.data.registrations[i].registrants.length; i2 < len2; i2++) {
+            spouseRegistrantRegistered = _.contains(_.map($scope.spouseRegistration.registrants, 'email'), response.data.registrations[i].registrants[i2].email);
+            if (spouseRegistrantRegistered) {
+              break;
+            }
+          }
+          if (spouseRegistrantRegistered) {
+            break;
+          }
+        }
+
+        //Check if current registrant already registered
+        for (var i3 = 0, len3 = response.data.registrations.length; i3 < len3; i3++) {
+          currentRegistrantCompleted = response.data.registrations[i3].completed;
```

Comment:
Isn't this variable redundant? It looks like `spouseRegistrantCompleted` will always match `currentRegistrantCompleted` because the logic for calculating both of those values is the same.